docs/docs/api-reference/core/atom.md

```diff
@@ -63,3 +63,7 @@ function Counter() {
   );
 }
 ```
+
+## Scoped Atoms
```

Comment:
I would title this section "Atom Families" and say "scoping your atom state or storing a collection of related states." or something like that.